3.7.1 fails to compile on SLES11

3.7.1 fails to compile on SLES11 with the following message:
```
[  175s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-afunix-source.lo
[  175s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-afunix-dest.lo
[  175s] /usr/include/./libnet/libnet-functions.h:1433: warning: inline function 'libnet_getgre_length' declared but never defined
[  175s] /usr/include/./libnet/libnet-functions.h:1433: warning: inline function 'libnet_getgre_length' declared but never defined
[  176s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-transport-mapper-unix.lo
[  176s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-transport-unix-socket.lo
[  176s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-unix-credentials.lo
[  176s]   CC     modules/afsocket/modules_afsocket_libafsocket_la-afsocket-grammar.lo
[  176s] modules/afsocket/transport-mapper-unix.c:38: error: redefinition of typedef 'TransportMapperUnix'
[  176s] modules/afsocket/transport-mapper-unix.h:28: error: previous declaration of 'TransportMapperUnix' was here
[  176s] make[2]: *** [modules/afsocket/modules_afsocket_libafsocket_la-transport-mapper-unix.lo] Error 1
[  176s] make[2]: *** Waiting for unfinished jobs....
[  177s] make[1]: *** [all-recursive] Error 1
[  177s] make: *** [all] Error 2
```

--- a/modules/afsocket/transport-mapper-unix.c
+++ b/modules/afsocket/transport-mapper-unix.c
@@ -30,12 +30,12 @@
 #include <sys/un.h>
 
 
-typedef struct _TransportMapperUnix
+struct _TransportMapperUnix
 {
   TransportMapper super;
   gchar *filename;
   gboolean pass_unix_credentials;
-} TransportMapperUnix;
+};
 
 static LogTransport*
 _create_log_transport(TransportMapper *s, gint fd)

